app/assets/javascripts/index/directions/graphhopper.js

```diff
@@ -79,5 +80,6 @@ function GraphHopperEngine(id, vehicleType) {
   };
 }
 
-OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_bicycle", "bike"), true);
-OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_foot", "foot"), true);
+OSM.Directions.addEngine(new GraphHopperEngine("graphhopper_car", "car", true), true);
```

Comment:
I assume that `disableCH` refers to contraction hierarchies, in which case I would have thought long distance car routing would be exactly the case where you would want them so I'm surprised that you are disabling them for car routing but leaving them enabled for other things?